templates/pages/error.html

```diff
@@ -43,9 +43,8 @@ <h1 class="title">{{ _("error.unexpected") }}</h1>
 
     {% if details %}
       <hr />
-      <pre>
-        <code class="font-mono whitespace-pre-wrap break-all">{{ details }}</code>
-      </pre>
+      {# caution: do not introduce whitespace between <pre> and <code> #}
```

Comment:
Good catch, I've noticed that but it never bothered me enough to actually fix it :D